src/components/hierarchy-bar/hierarchy-bar-standard/hierarchy-bar-standard.component.ts

```diff
@@ -64,20 +68,54 @@ export class HierarchyBarStandardComponent implements OnDestroy {
         // emit whether we are overflowing or not
         this.isOverflowing$.next(isOverflowing);
 
-        // if the hierarchy bar contents do not overflow then do nothing
-        if (isOverflowing) {
+        // we don't need to do anything else if there is no overflow
```

Comment:
Nothing major here, just that it may be better to keep the comments in the same style as the ones above? 

/** EXAMPLE */ as opposed to // EXAMPLE.

Ignore if this is a non issue.

Reply:
`/** comment */` is a documentation comment, and should only be used above public API elements such as classes, public inputs and outputs, public functions, etc. The code here is correct.

Reply:
Ah thanks for clearing that up!